components/Slider/src/MDCSlider.m

```diff
@@ -526,6 +545,12 @@ - (BOOL)accessibilityActivate {
 - (void)thumbTrackValueChanged:(__unused MDCThumbTrack *)thumbTrack {
   [self sendActionsForControlEvents:UIControlEventValueChanged];
   UIAccessibilityPostNotification(UIAccessibilityAnnouncementNotification, self.accessibilityValue);
+  if (@available(iOS 10.0, *)){
+    if (self.hapticsEnabled && (_thumbTrack.value == _thumbTrack.minimumValue || _thumbTrack.value == _thumbTrack.maximumValue)){
+      [self.feedbackGenerator prepare];
```

Comment:
from reading the docs, it seems that calling prepare and thereafter impactOccurred straight away doesn't reduce the latency that prepare provides. Is this more set for the following impacts that may occur by the user sliding back and forth? If so, do you feel it may be better to invoke prepare in thumbTrackTouchDown ?